### PR TITLE
feat: `MenuFlyout` support for keyboard accelerators

### LINKED ISSUE #15195: [Gallery][WASM/Android/iOS]MenuBar-  On the opened Menu flyout, 'Control key' names are missing for the 'Menu Bar with keyboard accelerators' option.
### Current behavior

Menubar issue

### Expected behavior

menubar correct

### How to reproduce it (as minimally and precisely as possible)

Open Uno Gallery APP
Tap Burger Menu on the left top right.
Tap "DatePicker" from the "Ui Component" list.
Go to 'MenuBar with keyboard accelerators'.
Click on 'File', 'Edit' or 'Help' menu button.
Notice that in the opened flyout the control key display option are missing.

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

WebAssembly, Android, iOS

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### LINKED ISSUE #17133: `MenuFlyout` and `ToggleMenuFlyoutItem` support for Keyboard Accelerators

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): closes #17133, closes #15195

## PR Type

What kind of change does this PR introduce?

- Feature

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/ParentVisualTreeListener.cs [hunk 1]
```diff
@@ -10,12 +10,15 @@ namespace Uno.UI.Xaml;
...
+	private readonly Action _onUnloaded;
 	private readonly SerialDisposable _registrations = new();
 
-	public ParentVisualTreeListener(DependencyObject owner)
+	public ParentVisualTreeListener(DependencyObject owner, Action onLoaded, Action onUnloaded)
 	{
 		_owner = owner;
-
+		_onLoaded = onLoaded;
+		_onUnloaded = onUnloaded;
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/ParentVisualTreeListener.cs:21]: Could this cause memory leaks? We may need additional leak testing for this.

File: src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_MenuFlyout.cs [hunk 2]
```diff
@@ -273,7 +273,7 @@ public async Task When_MenuFlyoutItem_CommandChanging()
 
 			await WindowHelper.WaitForLoaded(flyoutItem);
 
-			flyoutItem.InvokeClick();
+			flyoutItem.Invoke();
```
Comment (jeromelaban) [inline src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_MenuFlyout.cs:276]: Do we need additional manual testing for those?

File: src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarButton.Partial.cs [hunk 3]
```diff
@@ -882,5 +882,7 @@ private void PutKeyboardAcceleratorText(string keyboardAcceleratorText)
 		{
 			element = GetTemplateChild(name) as T;
 		}
+
+		public void SetSubMenuDirection(bool isSubMenuDirectionUp) { }
```
[the remark this answers is not in the document]
Comment (MartinZikmund) [reply src/Uno.UI/UI/Xaml/Controls/AppBar/AppBarButton.Partial.cs:886]: Yes, it is a noop, but the method comes from interface (which should be implemented explicitly, but that will be breaking)

File: src/Uno.UI/UI/Xaml/ParentVisualTreeListener.cs (hunk 1 above)
Comment (MartinZikmund) [reply src/Uno.UI/UI/Xaml/ParentVisualTreeListener.cs:21]: Yes, will have to improve that still

File: src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_MenuFlyout.cs (hunk 2 above)
Comment (MartinZikmund) [reply src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_MenuFlyout.cs:276]: This should  be equivalent, the method was just named differently

File: src/Uno.UI/UI/Xaml/ParentVisualTreeListener.cs (hunk 1 above)
Comment (MartinZikmund) [reply src/Uno.UI/UI/Xaml/ParentVisualTreeListener.cs:21]: @jeromelaban is there a blocker for https://github.com/unoplatform/uno/pull/15540?

**Comment by MartinZikmund** [comment]
Replaced by https://github.com/unoplatform/uno/pull/17292

**Pull request closed without merging**